example/ck_tile/17_grouped_gemm/grouped_gemm_preshuffle.cpp

```diff
@@ -167,6 +167,115 @@ float grouped_gemm(const std::vector<grouped_gemm_kargs>& gemm_descs,
     return ave_time = BaseGemmPipeline::TailHandler(RunSplitk, has_hot_loop, tail_num);
 }
 
+template <typename GemmConfig,
+          typename ALayout,
+          typename BLayout,
+          typename CLayout,
+          typename ADataType,
+          typename BDataType,
+          typename AccDataType,
+          typename CDataType>
+float grouped_gemm_tileloop(const ck_tile::stream_config& s,
+                            const ck_tile::index_t num_groups,
+                            void* kargs_ptr,
+                            bool splitk = false)
+{
+    using GemmShape = ck_tile::TileGemmShape<
+        ck_tile::sequence<GemmConfig::M_Tile, GemmConfig::N_Tile, GemmConfig::K_Tile>,
+        ck_tile::sequence<GemmConfig::M_Warp, GemmConfig::N_Warp, GemmConfig::K_Warp>,
+        ck_tile::
+            sequence<GemmConfig::M_Warp_Tile, GemmConfig::N_Warp_Tile, GemmConfig::K_Warp_Tile>>;
+    using TilePartitioner =
+        ck_tile::GemmSpatiallyLocalTilePartitioner<GemmShape,
+                                                   GemmConfig::TileParitionerGroupNum,
+                                                   GemmConfig::TileParitionerM01>;
+
+    using GemmUniversalTraits =
+        ck_tile::TileGemmUniversalTraits<GemmConfig::kPadM,
+                                         GemmConfig::kPadN,
+                                         GemmConfig::kPadK,
+                                         GemmConfig::DoubleSmemBuffer,
+                                         ALayout,
+                                         BLayout,
+                                         CLayout,
+                                         GemmConfig::TransposeC,
+                                         GemmConfig::UseStructuredSparsity,
+                                         false, // Persistent = false (not supported with preshuffle
```

Comment:
derive from GemmConfig
```suggestion
                                         GemmConfig::Persistent,
```

Reply:
Ack